src/reanimated2/animation/util.ts

```diff
@@ -43,6 +44,27 @@ if (__DEV__ && IS_REDUCED_MOTION) {
   );
 }
 
+const SHOULD_BE_USE_WEB = shouldBeUseWeb();
+
+export function assertEasingIsWorklet(
+  easing: EasingFunction | EasingFunctionFactory
+) {
+  'worklet';
+  if (_WORKLET) {
+    // If 'assertEasingIsWorklet' is called on UI runtime then easing is a copy and it is not a worklet
+    return true;
+  }
+  if (SHOULD_BE_USE_WEB) {
+    // It is possible to run reanimated on web without plugin, so let's skip this check on web
+    return true;
```

Comment:
This function just asserts so there's no need for it to have a return value.
```suggestion
    return;
```